Introduction.md

```diff
@@ -381,9 +381,11 @@ To do that, turn on the target membership in respective framework as set the mem
 This will ensure the relevant header file is exported from WTF / WebCore to other downstream projects like WebKitLegacy.
 ![Screenshot of exporting a header file](resources/xcode-export-header.png)
 
+WPE and GTK use CMake for building the project. If the header is not platform-specific, you might want to add an entry for it into
```

Comment:
Can you make the "might" more specific? Is this only necessary if e.g. a WebCore header is included by WebKit?

Reply:
It is only necessary if its included by WebKit.

Also it isn't just WPE and GTK using CMake. Every non-Cocoa port uses it. I'd say non-Apple but AppleWin uses CMake.

Reply:
@donny-dont  How about WebKit/Headers.cmake ?